scheduler/test/cook/test/mesos/api.clj

```diff
@@ -554,6 +554,15 @@
             get-body (response->body-data get-resp)]
         (is (= get-body update-body))))
 
+    (testing "quota is checked on job submission"
+      (let [job (assoc (basic-job) "cpus" 32.0)
+            job-resp (h (merge quota-req-attrs
+                               {:uri "/rawscheduler"
+                                :authorization/user "foo"
+                                :request-method :post
+                                :body-params {"jobs" [job]}}))]
+        (is (= 422 (:status job-resp)))))
+
```

Comment:
Doesn't the integration test cover this? Are we getting any additional value out of the unit test?

Reply:
I added this because I was debugging locally and this was much faster than doing integration tests, but now that it's stable it's just redundant. I'll remove it.

Reply:
I deleted the redundant CPUs test, but it occurred to me that we can test GPUs here (which are not being tested in the integration tests), so now this is a GPU-quota test.